src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,19 @@
 import React from 'react';
 import './MoviesList.scss';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Movies = {
```

Comment:
This type better call "Props" because you have  lot of movie vars.
![photo_2023-03-11_18-46-58](https://user-images.githubusercontent.com/105153156/224496878-487a09df-d66d-47e3-815c-e947d6cd2b0e.jpg)